### PR TITLE
Creating simple RF dummy test that runs with Docker

### PR DESCRIPTION
**Pull request by 2laJ2**
Before web testing can be done with robot, downloading and installing web browser and browser driver must be added into the Dockerfile. The hello_world.robot contains only a simple dummy test to test if its run correctly in container.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,12 @@ FROM ubuntu
 
 RUN apt update && apt install -y git make unzip curl g++ python3 pip wget
 COPY /build/requirements.txt ./
+COPY hello_world.robot ./robot/hello_world.robot
+
+RUN python3 -m pip install robotframework
```
Comment (doyu) [inline Dockerfile:7]: Shouldn't they be in "requirements.txt"?

**Review by doyu — COMMENTED** [review]
Can CI test is passed?

**Comment by 2laJ2** [comment]
If I make an image and run it, both tests (the one from tensorflow and hello_world.robot) pass.

If I remove lines 7,8,9 and add lines "robotframework", "robotframework-seleniumlibrary" and "selenium" to requirements.txt and then run command "pip3 install -r requirements.txt" and "robot hello_world.robot" in the terminal, the test hello_world.robot will pass.

But if I make the image and run it, I get this error message: 
 
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "robot": executable file not found in $PATH: unknown

So it seems that there is a problem in finding the path to hello_world.robot? How can I fix it? How can I put the line 5 (COPY hello_world.robot ./robot/hello_world.robot) into the requirements.txt?

**Comment by doyu** [comment]
@2laJ2 Can you make a PR with the latest changes? And try to look for some help in Discord, just in case.

With that PR, people could try easily to reproduce your issue

**Comment by 2laJ2** [comment]
Here are the new files Dockerfile and requirements.txt. So the problem is as described above: when I build the image and then try to run the container, I get the error message. so how can this be fixed?

**Comment by doyu** [comment]
Does the latest change work OK? Or do you still have any problem? If so please describe what the problem is here. Thanks!

**Comment by FexbYk23** [comment]
The problem is that you've added the dependencies to the requirements.txt used by the web app. The hello world uses build/requirements.txt.

**Comment by doyu** [comment]
Wouldn't it be better to introduce dedicate dir per WebApp and Hello respectively?

**Comment by 2laJ2** [comment]
Would you like to clarify - do I make separate folder in the repo for robot and then put the test file and Dockerfile for robot there? So that RF has it's separate Dockerfile? The container was working fine, until the installation of RF was put in requirements.txt. I am not able to find a way to make this combined Dockerfile to run with these modifications I was asked to make. So do I now go back to the test file and Dockerfile for RF only and put them into another folder?

File: Dockerfile [hunk 2]
```diff
@@ -5,6 +5,7 @@ COPY /build/requirements.txt ./
 COPY hello_world.robot ./robot/hello_world.robot
 
 RUN pip3 install -r requirements.txt
+
```
Comment (doyu) [inline Dockerfile:8]: unnecesasry change?

**Comment by doyu** [comment]
This is a part of https://github.com/Origami-TinyML/tflm_hello_world/issues/9

File: Dockerfile (hunk 2 above)
Comment (2laJ2) [reply Dockerfile:8]: Must have been a typo :)

**Pull request merged**